### PR TITLE
CloudXNS Support

### PR DESCRIPTION
**Pull request by zh99998**
yet another big DNS service in China.

### CODE REVIEW

**Comment by CapsuleCD** [comment]
Hi.

I'm an automated pull request bot named [CapsuleCD](http://www.github.com/AnalogJ/capsulecd). I handle testing, versioning and package releases for this project. 
- If you're the owner of this repo, you can click the button below to kick off the CapsuleCD build pipeline and create an automated release.'
- If not, don't worry, someone will be by shortly to check on your pull request. 

[CapsuleCD](http://www.capsulecd.com/#!/deploy/github/AnalogJ/lexicon/pullrequests/42)

---
If you're interested in learning more about [CapsuleCD](http://www.github.com/AnalogJ/capsulecd), or adding it to your project, you can check it out [here](http://www.github.com/AnalogJ/capsulecd)

**Comment by AnalogJ** [comment]
Hey @zh99998, can you include the working test fixtures for CloudXNS? Use a fake domain or `capsulecd.com` if you can. 

You can run the tests and then add the fixtures:

    py.test tests/providers/test_cloudxns.py
    git add tests/fixtures/cassettes/cloudxns/IntegrationTests/test_*.yaml

**Comment by zh99998** [comment]
need help

```
➜  lexicon-docker git:(master) py.test tests/providers/test_cloudxns.py
============================================================================= test session starts ==============================================================================
platform linux2 -- Python 2.7.6 -- pytest-2.5.1
collected 0 items / 1 errors 
            
==================================================================================== ERRORS ====================================================================================
______________________________________________________________ ERROR collecting tests/providers/test_cloudxns.py _______________________________________________________________
tests/providers/test_cloudxns.py:3: in <module>  
>   from integration_tests import IntegrationTests
tests/providers/integration_tests.py:10: in <module>
>           decode_compressed_response=True
E   TypeError: __init__() got an unexpected keyword argument 'decode_compressed_response'
=========================================================================== 1 error in 0.16 seconds ============================================================================
```

version info:
```
➜  lexicon-docker git:(master) pip -V
pip 8.1.2 from /usr/local/lib/python2.7/dist-packages (python 2.7)
➜  lexicon-docker git:(master) python -V
Python 2.7.6
➜  lexicon-docker git:(master) pip show vcrpy
---
Metadata-Version: 1.1
Name: vcrpy
Version: 1.7.4
Summary: Automatically mock your HTTP interactions to simplify and speed up testing
Home-page: https://github.com/kevin1024/vcrpy
Author: Kevin McCarthy
Author-email: me@kevinmccarthy.org
License: MIT
Location: /usr/local/lib/python2.7/dist-packages
Requires: PyYAML, wrapt, six, contextlib2, mock, contextlib2, mock
Classifiers:
  Development Status :: 4 - Beta
  Environment :: Console
  Intended Audience :: Developers
  Programming Language :: Python
  Programming Language :: Python :: 3
  Topic :: Software Development :: Testing
  Topic :: Internet :: WWW/HTTP
  License :: OSI Approved :: MIT License
```

File: tests/providers/test_cloudxns.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Test for one implementation of the interface
+from lexicon.providers.dnspod import Provider
```
Comment (AnalogJ) [inline tests/providers/test_cloudxns.py]: This line should be `from lexicon.providers.cloudxns import Provider`

Comment (zh99998) [inline tests/providers/test_cloudxns.py]: still
```
➜  lexicon-docker git:(master) cat tests/providers/test_cloudxns.py
# Test for one implementation of the interface
from lexicon.providers.cloudxns import Provider
from integration_tests import IntegrationTests
from unittest import TestCase
import pytest

# Hook into testing framework by inheriting unittest.TestCase and reuse
# the tests which *each and every* implementation of the interface must
# pass, by inheritance from define_tests.TheTests
class DnsParkProviderTests(TestCase, IntegrationTests):

    Provider = Provider
    provider_name = 'cloudxns'
    domain = 'capsulecd.com'
    def _filter_post_data_parameters(self):
        return ['login_token']
➜  lexicon-docker git:(master) py.test tests/providers/test_cloudxns.py
============================================================================= test session starts ==============================================================================
platform linux2 -- Python 2.7.6 -- pytest-2.5.1
collected 0 items / 1 errors 

==================================================================================== ERRORS ====================================================================================
______________________________________________________________ ERROR collecting tests/providers/test_cloudxns.py _______________________________________________________________
tests/providers/test_cloudxns.py:3: in <module>
>   from integration_tests import IntegrationTests
tests/providers/integration_tests.py:10: in <module>
>           decode_compressed_response=True
E   TypeError: __init__() got an unexpected keyword argument 'decode_compressed_response'
=========================================================================== 1 error in 0.21 seconds ============================================================================
```

**Pull request merged**